README.md

```
@@ -4,103 +4,83 @@ This software is an SMS Bridge Consolidator that works on a laptop connected to
+3. **Verify**: Check that services are running
+   ```bash
+   docker-compose ps
+   ```
+   
+   Login to Admin UI at: http://localhost:8080/admin/
 
 ## Service Access
 
-Both deployment methods provide access to:
+Default service endpoints:
 - **SMS Receiver**: http://localhost:8080
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)